drivers/volume/portworx/portworx.go

```diff
@@ -1228,6 +1228,17 @@ func (d *portworx) GetReplicaSetNodes(torpedovol *torpedovolume.Volume) ([]strin
 	return pxNodes, nil
 }
 
+func (d *portworx) UpdateNodeID(n node.Node) (node.Node, error) {
```

Comment:
@thiguetta instead of an explicit UpdateNodeID, it would be safer to always looking up the node ID every time the portworx driver needs to perform an operation based on ID.